charts/consul/templates/crd-ingressgateways.yaml

```diff
@@ -78,6 +78,40 @@ spec:
                       while waiting for a connection to be established.
                     format: int32
                     type: integer
+                  passiveHealthCheck:
+                    description: PassiveHealthCheck configuration determines how upstream
+                      proxy instances will be monitored for removal from the load
+                      balancing pool.
+                    properties:
+                      baseEjectionTime:
+                        description: The base time that a host is ejected for. The
+                          real time is equal to the base time multiplied by the number
+                          of times the host has been ejected and is capped by max_ejection_time
+                          (Default 300s). Defaults to 30000ms or 30s.
+                        type: string
+                      enforcingConsecutive5xx:
+                        description: EnforcingConsecutive5xx is the % chance that
+                          a host will be actually ejected when an outlier status is
+                          detected through consecutive 5xx. This setting can be used
+                          to disable ejection or to ramp it up slowly.
+                        format: int32
+                        type: integer
+                      interval:
```

Comment:
How is this string to be formatted?

Reply:
I'm not sure what would actually apply here: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#format
I think this is just "basic string"

Reply:
For this, I would add an example. I think the answer is something like "10s" as the way the input should be formatted.

Reply:
done